src/schedule/schedule-week-view/schedule-week-view.tsx

```diff
@@ -0,0 +1,186 @@
+import {
+    BlankCell,
+    BodyContainer,
+    Description,
+    HeaderContainer,
+    LoadingContainer,
+    ServiceContainer,
+    ServiceHeader,
+    SlotColumn,
+    Timeline,
+    SlotGrid,
+} from "./schedule-week-view.styles";
+import { ThemedLoadingSpinner } from "../../animations/themed-loading-spinner/themed-loading-spinner";
+import { TimeCell } from "./time-cell";
+
+// =============================================================================
+// MAIN COMPONENT
+// =============================================================================
+
+export const ScheduleWeekView = ({
+    date,
+    serviceData,
+    loading,
+    minTime,
+    maxTime,
+    initialScrollTime,
+    containerRef,
+    blockedMessage,
+    onClickHiddenSlots,
+}: ScheduleWeekViewProps) => {
+    // =============================================================================
+    // CONST, STATE, REF
+    // =============================================================================
+    const timelineOffset = useTimelineOffset(minTime, maxTime);
+    const bodyRef = useInitialScroll(loading, minTime, initialScrollTime);
+    const [hoverDay, setHoverDay] = useState<dayjs.Dayjs>();
+
+    const weekDays = useMemo(() => {
+        const startOfWeek = dayjs(date).startOf("week");
+        return Array.from({ length: 7 }, (_, i) => startOfWeek.add(i, "day"));
+    }, [date]);
+
+    const timeSlots = useMemo(() => {
+        return TimeHelper.generateTimings(30, "24hr", minTime, maxTime);
+    }, [minTime, maxTime]);
+
+    const slotLayoutMap = useMemo(() => {
+        return calculateSlotWidths(serviceData, weekDays, timeSlots);
+    }, [serviceData, weekDays, timeSlots]);
+
+    const today = dayjs();
+
+    // =============================================================================
+    // EVENT HANDLERS
+    // =============================================================================
+    const handleDayHover = (value: dayjs.Dayjs) => {
+        setHoverDay(value);
+    };
+
+    const handleDayMouseout = () => {
+        setHoverDay(undefined);
+    };
+
+    // =============================================================================
+    // HELPER FUNCTIONS
+    // =============================================================================
+    const generateStyleProps = (day: dayjs.Dayjs): CellStyleProps => {
+        const isToday = day.isSame(today, "day");
+        const isHover = hoverDay && day.isSame(hoverDay, "day");
+
+        const dayCellStyleProps: CellStyleProps = {
+            labelType: isToday ? "current" : "available",
+            interactive: null,
+        };
+
+        if (isToday) {
+            dayCellStyleProps.circleLeft = "selected-outline";
+            dayCellStyleProps.circleRight = "selected-outline";
+        }
+        if (isHover && !isToday) {
+            dayCellStyleProps.circleLeft = "hover-subtle";
+            dayCellStyleProps.circleRight = "hover-subtle";
```

Comment:
actually since this is only used to indicate the current date, and user can't change the selection, we don't need the hover style? since there's no interactivity